pyseir/inference/infer_rt.py

```diff
@@ -543,24 +675,37 @@ def infer_all(self, plot=True, shift_deaths=0):
             df_all["Rt_MAP_composite"] = df_all["Rt_MAP__new_cases"]
             df_all["Rt_ci95_composite"] = df_all["Rt_ci95__new_cases"]
 
+        for i in range(0, InferRtConstants.SMOOTH_RT_MAP_COMPOSITE):
+            # Optionally Smooth just Rt_MAP_composite. Note this doesn't lag in time and preserves
+            # integral of Rteff over time
+            kernel_width = round(InferRtConstants.RT_SMOOTHING_WINDOW_SIZE / 4)
+            smoothed = (
+                df_all["Rt_MAP_composite"]
+                .rolling(
+                    InferRtConstants.RT_SMOOTHING_WINDOW_SIZE,
+                    win_type="gaussian",
+                    min_periods=kernel_width,
+                    center=True,
+                )
+                .mean(std=kernel_width)
+            )
+            df_all["Rt_MAP_composite"] = smoothed
+            df_all["Rt_ci95_composite"] = (
+                (df_all["Rt_ci95_composite"] - df_all["Rt_MAP_composite"])
+                / math.sqrt(
+                    2.0 * kernel_width  # averaging over many points reduces confidence interval
+                )
+                + df_all["Rt_MAP_composite"]
+            )
+
         if plot and df_all is not None:
             plt.figure(figsize=(10, 6))
 
-            if "Rt_MAP_composite" in df_all:
-                plt.scatter(
-                    df_all.index,
-                    df_all["Rt_MAP_composite"],
-                    alpha=1,
-                    s=25,
-                    color="yellow",
-                    label="Inferred $R_{t}$ Web",
-                    marker="d",
-                )
-            plt.hlines([1.0], *plt.xlim(), alpha=1, color="g")
-            plt.hlines([1.1], *plt.xlim(), alpha=1, color="gold")
-            plt.hlines([1.3], *plt.xlim(), alpha=1, color="r")
+            # plt.hlines([1.0], *plt.xlim(), alpha=1, color="g")
+            # plt.hlines([1.1], *plt.xlim(), alpha=1, color="gold")
+            # plt.hlines([1.3], *plt.xlim(), alpha=1, color="r")
 
-            if "Rt_ci5__new_deaths" in df_all:
+            if "Rt_ci5__new_deaths" in df_all and not InferRtConstants.DISABLE_DEATHS:
```

Comment:
I removed it because the chart is very busy with all that detail on it - especially all the overlapping confidence intervals. I made other changes to the chart as well in terms of scale and colors for the final composite to improve readability. None of that makes it into any of our charting on the web of course. Do you think this information would be useful for comparison purposes?